docs/community/modules/chat/pages/index.adoc

```diff
@@ -1,17 +1,19 @@
 = Chat
 
-The Jenkins project meets to discuss various topics on Gitter and IRC.
+The Jenkins project meets to discuss various topics on Matrix, commonly referenced as “Gitter,” because most of our rooms live on the gitter.im server..
```

Comment:
```suggestion
The Jenkins project meets to discuss various topics on Matrix, commonly referenced as “Gitter,” because most of our rooms live on the gitter.im server.
```